content/300-guides/200-deployment/201-serverless/300-deploy-to-vercel.mdx

```diff
@@ -292,29 +57,18 @@ In this section you associate a second PostgreSQL database with a Vercel preview
 
    ![Add an environment variable for the preview environment](./images/300-60-deploy-to-vercel-preview-environment-variable.png)
 
-4. Set the value to the connection string of your second PostgreSQL database. As before, the connection string has the following format:
+4. Set the value to the connection string of your second database. As a reminder, a PostgreSQL connection string has the following format:
```

Comment:
```suggestion
4. Set the value to the connection string of your second database:
```

Also, It would be worth using an example connection string with placeholder values rather than highlighting the format, i.e. using `__USER__`. (similar to what we use for blog posts)